.github/workflows/integration-tests.yaml

```diff
@@ -116,9 +131,27 @@ jobs:
             --set gitea.admin.password=${GITEA_ADMIN_PASSWORD} \
             --wait
 
+      - name: Check Helm repo for image
+        id: helm_repo_check
+        shell: bash
+        env:
+          KEPTN_VERSION: ${{ matrix.keptn-version }}
+        run: |
+          echo "Keptn Version: $KEPTN_VERSION"
+          helm repo add keptn https://charts.keptn.sh
+          helm_search=$(helm search repo keptn/keptn --version $KEPTN_VERSION)
+          
+          if [[ "$helm_search" != "No results found" ]]; then
+          echo "Using standard Helm repo"
+          echo "::set-output name=HELM_REPO::https://charts.keptn.sh"
+          else
+          echo "Using dev Helm repo"
+          echo "::set-output name=HELM_REPO::https://charts-dev.keptn.sh"
+          fi
+
       - name: Install Keptn
         id: install_keptn
-        uses: keptn-sandbox/action-install-keptn@v2.0.0
+        uses: keptn-sandbox/action-install-keptn@main
```

Comment:
I think that we should pin a version here to avoid breaking things in the future if a new major version is released

Reply:
Yes, we didn't have a new release for the Helm repository changes, but we will release a new version after fixing the Helm values for Keptn 0.17. After the release, I will define the version here.

Reply:
I just released a new version and added it here.